src/components/TradeWidget/utils.ts

```diff
@@ -28,13 +28,19 @@ export function calculateReceiveAmount(priceValue: string, sellValue: string, re
   return receiveAmount
 }
 
-export const preprocessTokenAddressesToAdd = (addresses: (string | undefined)[], networkId: number): string[] => {
-  const tokenAddresses: string[] = []
+export const preprocessTokenAddressesToAdd = (
+  addresses: (string | undefined)[],
+  networkId: number,
+): (string | null)[] => {
+  const tokenAddresses: (string | null)[] = []
   const addedSet = new Set()
 
   addresses.forEach((address) => {
+    if (!address) {
+      tokenAddresses.push(null)
```

Comment:
Why does this need an explicit null push?

Reply:
You are right, doesn't need `null`
I added it for positional tokens, to know which added token is for sell,which for receive, but then I opted for an extra variable to keep track of them `positionedAddedTokens`. Forgot to remove